daemon/algod/api/generate.sh

```diff
@@ -4,20 +4,19 @@ set -e
 rootdir=`dirname $0`
 pushd ${rootdir}
 
-cd ${rootdir}/../../..
-$(GO111MODULE=on go get -u "github.com/algorand/oapi-codegen/...@v1.3.5-algorand4")
-
+pushd ${rootdir}/../../..
+  $(GO111MODULE=on go get -u "github.com/algorand/oapi-codegen/...@v1.3.5-algorand4")
 popd
 
 # Convert v2 to v3
 curl -s -X POST "https://converter.swagger.io/api/convert" -H "accept: application/json" -H "Content-Type: application/json" -d @./algod.oas2.json  -o 3.json
-cat 3.json | json_pp > algod.oas3.yml
+
+# Sort keys, format json and rename 3.json -> algod.oas3.yml
+python3 -c "import json; import sys; json.dump(json.load(sys.stdin), sys.stdout, indent=2, sort_keys=True)" < 3.json > algod.oas3.yml
```

Comment:
I think that `pq` is a better tool here, but that's fine either.

Reply:
Interesting, I didn't know about that one. It's not installed on my machine though.

Brian just deprecated this script with a Makefile, so I'm going to revert these changes and avoid the merge conflict.